hardware/computemodule/cmio-camera.md

```diff
@@ -1,12 +1,28 @@
 # Attaching a Raspberry Pi Camera Module to the Compute Module IO Board
 
-**This document is a work in progress and is intended for advanced users.**
+## Notes before starting
 
-**For the camera to work with the Compute Module 3 (CM3), the firmware needs to be September 21st 2016 or newer (use `vcgencmd version` to check).**
+These instructions are intended for expert users, if anything is unclear please use the [Raspberry Pi Camera forums](https://www.raspberrypi.org/forums/viewforum.php?f=43) for technical help. 
 
-**Note for designers attaching Camera Modules directly to CM carrier boards:** it is NOT necessary to incorporate the crypto chip used on the Raspberry Pi–designed camera boards when attaching the OM5647 or IMX219 Camera Modules directly to the CM carrier board. The Raspberry Pi firmware will automatically detect the CM and allow communications with the Camera Module to proceed without the crypto chip being present.
+Unless explicitly stated otherwise, these instructions will work identically on Compute Module and Compute Module 3 Module+IO board(s).
```

Comment:
`on Compute Module and Compute Module 3 Module+IO board(s)` might be better as something like: 

`on both the Compute Module and Compute Module 3, attached to a Compute Module IO Board`?